src/order/filter/lift.lean

```diff
@@ -23,12 +25,47 @@ protected def lift (f : filter α) (g : set α → filter β) :=
 
 variables {f f₁ f₂ : filter α} {g g₁ g₂ : set α → filter β}
 
+/-- If `{s i | p i, i : ι}` is a basis of a filter `f`, `g` is a monotone function
+`set α → filter γ`, and for each `i`, `{sg x | pg x, x : β i}` is a basis of the filter `g (s i)`,
```

Comment:
Is `{sg x | pg x, x : β i}` now valid syntax?

Reply:
AFAIK no but I thought that we can use non-valid Lean in comments. If you think otherwise, then I'll try to reformulate.

Reply:
I think using non-Lean which looks like Lean is confusing, but I won't insist.

Reply:
This notation confused me as well. You may as well use the actually valid version of this, `{(s i) | (i : ι) (_ : p i)}`, since it's approximately as expressive and doesn't require invented notation.

Reply:
I tried to write better docstrings.